erpnext/projects/doctype/task/task.json

```diff
@@ -112,6 +112,7 @@
   },
   {
    "bold": 1,
+   "description": "the status of the default project will fetch from the projects table.",
```

Comment:
It should be like .... "**T**he......default **P**roject........**P**rojects table". 
As the label of the table is Projects and Project is also a DocType.